src/inc/msquic.hpp

```diff
@@ -421,6 +448,17 @@ class MsQuicConfiguration {
                 KeyCount * sizeof(QUIC_TICKET_KEY_CONFIG),
                 KeyConfig);
     }
+    QUIC_STATUS
+    SetSettings(_In_ const MsQuicSettings& Settings) noexcept {
+        const QUIC_SETTINGS* QSettings = &Settings;
```

Comment:
Why the extra temporary variable?

Reply:
Because it lets us guarantee that sizeof that variable is the size MsQuic expects, even if we add extra variables to MsQuicSettings that increase its size.